source/style-guide/information-types/prototypes/reference/server-params-style.txt

```diff
@@ -0,0 +1,129 @@
+.. _meta-reference-server-params:
+
+=======================
+Server Parameters Style
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Details to Document
+-------------------
+
+When documenting a new server parameter, include the following details 
+in addition to specific information about the new parameter:
+
+.. list-table::
+   :widths: 40 40
+   :header-rows: 1
+
+   * - Documentation Detail
+     - Necessity
+     - Description
+
+   * - mongod / mongos Support
+     - Required
+     - Whether the parameter is supported on ``mongod``, ``mongos``, or both.
+       Use ``|mongod-only|``, ``|mongos-only|``, or ``|both|`` as appropriate.
+
+   * - Version the Parameter was Added or Changed In
+     - Optional
```

Comment:
Is this optional or conditional? Seems important when it applies.

Reply:
Conditional strikes me as more appropriate. Thanks!